docs-en/chapter_array_and_linkedlist/array.md

```diff
@@ -1,14 +1,14 @@
 # Arrays
 
-The "array" is a linear data structure that stores elements of the same type in contiguous memory locations. We refer to the position of an element in the array as its "index". The following image illustrates the main terminology and concepts of an array.
+An "array" is a linear data structure that stores elements of the same type in contiguous memory spaces. We refer to the position of an element in an array as its "index". The figure below illustrates the primary terminology and concepts of an array.
```

Comment:
Recommended: The position of an element in an array is referred to as its "index“.

The passive voice makes the sentence more objective.

Reply:
thank you so much for the review Yanedie. I have fixed whats been mentioned (not including the final review).
I will try simplify it on the next commit, it might take a bit of time. 
thanks and Happy new year!